### PR TITLE
Add option to disable content type caching

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## Master
+### Added
+* Added option to disable completely Content Type caching.
```
Comment (mariobodemann) [inline CHANGELOG.md]: My english-🧠 thinks  `completely` might belong to the end of the sentence. What do you think?

File: spec/lib/contentful/management/space_spec.rb [hunk 2]
```diff
@@ -50,6 +66,23 @@ module Management
             expect(space.default_locale).to eql 'en-US'
           end
         end
+        describe 'content type caching' do
+          it 'fetches all content types when finding a space' do
+            vcr('space/find') do
+              space = subject.find(space_id)
+              expect(client.dynamic_entry_cache).not_to be_empty
```
Comment (mariobodemann) [inline spec/lib/contentful/management/space_spec.rb]: Can you be more exact here? What space did it find here?

File: spec/lib/contentful/management/space_spec.rb [hunk 3]
```diff
@@ -50,6 +66,23 @@ module Management
...
+            end
+          end
+          it 'doesnt fetch content types when explicitly disabled' do
```
Comment (mariobodemann) [inline spec/lib/contentful/management/space_spec.rb]: Hmm, `doesnt` without the tick just looks strange to me. Do you mind changing it to `does not`?

File: CHANGELOG.md (hunk 1 above)
Comment (Roblinde) [inline CHANGELOG.md]: "Added option to completely disable Content Type caching"

**Comment by dlitvakb** [comment]
@mariobodemann updated

**Pull request merged**